pages/PreviewPage/index.js

```diff
@@ -0,0 +1,66 @@
+import { StyledButton } from "@/components/StyledButton";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import Header from "../../components/Header";
+import PreviewPicture from "../../components/PreviewPicture";
+import { saveAs } from "file-saver";
+import { css } from "styled-components";
+
+const imageSrc = "https://i.imgur.com/8nUXl6s.jpeg/";
+const imageName = "8nUXl6s.jpeg";
+
+export default function PreviewPage() {
+  const router = useRouter();
+  const imageID = "8nUXl6s";
+
+  function handleVariations(event) {
+    event.preventDefault();
```

Comment:
since this is not a form you don't have an event property. Also you don't need event.preventDefault(). And just to link to another page you can better use a Link.